packages/app/components/Queue/Student/StudentBanner.tsx

```diff
@@ -159,16 +162,16 @@ export default function StudentBanner({
             <>
               <LeaveQueueButton leaveQueue={leaveQueue} />
               {studentQuestion.isOnline && (
-                <BannerButton
-                  icon={<TeamOutlined />}
-                  onClick={async () => {
-                    await API.questions.update(studentQuestion.id, {
-                      status: OpenQuestionStatus.PriorityQueued,
-                    });
-                  }}
-                >
-                  Re-join Queue
-                </BannerButton>
+                <Tooltip title="Re-join Queue">
+                  <BannerButton
+                    icon={<TeamOutlined />}
```

Comment:
The `<TeamOutlined />` icon on the top of the banner didn't make it too clear where to I needed click to rejoin the queue. 

Using the `<UndoOutlined />` icon would probably help but if we want to make it super clear on how to rejoin the queue we could put the re-join queue button below the text in the banner itself.

Reply:
bet